models/topics-models.js

```diff
@@ -5,3 +5,20 @@ exports.selectTopics = () => {
     return result.rows;
   });
 };
+
+exports.selectTopic = (topic) => {
+  if (!/^[a-z-_]/.test(topic)) {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
```

Comment:
I see what you've done here, but I'd take this out, because what if someone did want a topic that was just numbers? If you look in the seed, the topics are set up so that VARCHAR is valid, meaning we could technically have a topic which is numbers 🙂